Pod/Classes/CodeAttributedString.swift

```diff
@@ -56,13 +56,6 @@ open class CodeAttributedString : NSTextStorage
     }
 
 	/// Initialize the CodeAttributedString
-	required public init(itemProviderData data: Data, typeIdentifier: String) throws
```

Comment:
@brunophilipe I saw you added this method earlier, but I no longer see this constructor any more on the macOS side. If this is required for iOS, let me know, I will add it back.